loadtest/loadtest.go

```diff
@@ -72,7 +72,7 @@ func (lt *LoadTester) AddUsers(numUsers int) (int, error) {
 		return 0, ErrNotRunning
 	}
 	for i := 0; i < numUsers; i++ {
-		if err := lt.addUser(); err != nil {
+		if err := lt.addUser(i); err != nil {
```

Comment:
This would result in starting an admin every time we add new users to the load-test.